source/static/js/download.js

```diff
@@ -0,0 +1,26 @@
+;
+(function(d, n) {
+  var os = n.platform.match(/(Win|Mac|Linux)/);
+  var x = n.userAgent.match(/x86_64|Win64|WOW64/) ||
+    n.cpuClass === 'x64' ? 'x64' : 'x86';
+  var base = 'https://iojs.org/dist/latest/';
+  var db = d.getElementById('home-downloadbutton');
```

Comment:
As I just added this on nodejs.org as well:

```js
if (!db) { return; }
```

So this script throws no error if it is included on a page without download button.